src/main/java/teammates/ui/template/FeedbackSessionPublishButton.java

```diff
@@ -15,11 +15,12 @@
     private String actionName;
     private String actionLink;
     private boolean actionAllowed;
+    private boolean showButtonAndTooltip;
```

Comment:
There is no need to add this variable. This property is not referenced anywhere in the tags and it is not a good idea to mix front-end styling with the back end template.